eu/intermodalics/tango_nodelet_manager/1.1.0/tango_nodelet_manager-1.1.0.pom

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>eu.intermodalics</groupId>
+  <artifactId>tango_nodelet_manager</artifactId>
+  <version>1.1.0</version>
+  <packaging>aar</packaging>
+  <dependencies>
+    <dependency>
+      <groupId>org.ros.rosjava_core</groupId>
+      <artifactId>rosjava</artifactId>
+      <version>[0.3, 0.4)</version>
```

Comment:
@PerrineAguiar for example here you are correctly depending on rosjava kinetic, unlike message_generation below.